docs/getting_started/faq.rst

```diff
@@ -54,7 +54,14 @@ with dataframes that don't fit into the available memory. As a result, pandas wo
 for prototyping on a few MBs of data do not scale to tens or hundreds of GBs (depending on the size
 of your machine). Modin supports operating on data that does not fit in memory, so that you can comfortably
 work with hundreds of GBs without worrying about substantial slowdown or memory errors. For more information,
-see :doc:`out-of-memory support <getting_started/out_of_core.rst>` for Modin.
+see :doc:`out-of-memory support <getting_started/why_modin/out_of_core>` for Modin.
+
+How does Modin compare to Dask DataFrame and Koalas?
+""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+TLDR: Modin has better coverage of the pandas API, has a flexible backend, better ordering semantics,
+and supports both row and column-parallel operations.
+Check out this :doc:`page <getting_started/why_modin/modin_vs_dask_vs_koalas>` detailing the differences!
```

Comment:
The link doesn't work. Please check all the links you modified.

Reply:
Figured out the problem -- and fixed other broken links in the documentation I found